baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ZeroWarmupRateLimiter.java

```diff
@@ -81,4 +90,9 @@ private static boolean isDurationZero(ExpressionTree expressionTree, VisitorStat
         }
         return false;
     }
+
+    private static boolean isIntLiteralZero(ExpressionTree expressionTree, VisitorState state) {
+        Integer warmupTime = ASTHelpers.constValue(expressionTree, Integer.class);
+        return warmupTime != null && warmupTime.equals(0);
+    }
```

Comment:
I wonder if this would be more robust using `Number` (entirely untested):

```suggestion
    private static boolean isLiteralZero(ExpressionTree expressionTree, VisitorState state) {
        Number warmupTime = ASTHelpers.constValue(expressionTree, Number.class);
        return warmupTime != null && warmupTime.longValue() == 0L;
    }
```